doc/source/release-notes/Percona-Server-5.6.36-82.0.rst

```diff
@@ -0,0 +1,83 @@
+.. rn:: 5.6.36-82.0
+
+============================
+|Percona Server| 5.6.36-82.0
+============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.36-82.0 on
+May 11, 2017 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.36-82.0/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.34
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-36.html>`_, including
+all the bug fixes in it, |Percona Server| 5.6.34-79.1 is the current GA release
+in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.6.36-82.0
+milestone at Launchpad
+<https://launchpad.net/percona-server/+milestone/5.6.36-82.0>`_.
+
+New Features
+============
+
+ |Percona Server| 5.6 packages are now available for Ubuntu 17.04 (*Zesty
+ Zapus*).
+
+ |Percona Server| now supports :ref:`prefix_index_queries_optimization`.
+
+ :variable:`tokudb_dir_cmd` can now be used to :ref:`edit the TokuDB
+ <editing_tokudb_files_with_tokudb_dir_cmd>` files.
+
+Bugs Fixed
+==========
+
+ |Percona Server| would hang during OLTP prepare with several buffer pool
+ instances and a small :variable:`innodb_log_file_size` value. Bug fixed
+ :bug:`1651657`.
+
+ Querying :table:`TABLE_STATISTICS` in combination with a stored function could
+ lead to a server crash. Bug fixed :bug:`1659992`.
+
+ :file:`tokubackup_slave_info` file was created for a master server after
+ taking the backup with :ref:`toku_backup`. Bug fixed :bkpbug:`135`.
+
+ Fixed a memory leak in :ref:`toku_backup`. Bug fixed :bug:`1669005`.
+
+ :ref:`compressed_columns` could not be added to a partitioned table by using
+ ``ALTER TABLE``. Bug fixed :bug:`1671492`.
+
+ Fixed a memory leak which happened that happened in case of failure to create
+ a multi-threaded slave worker thread. Bug fixed :bug:`1675716`.
+
+ Combination of using the :ref:`audit_log_plugin` with MB collation connection
+ and ``PREPARE`` statement with a parse error could lead to a server crash. Bug
+ fixed :bug:`1688698` (upstream :mysqlbug:`86209`).
+
+ Fix for a :bug:`1433432` bug in |Percona Server| rn:`5.6.28-76.1` caused a
+ performance regression. Bug fixed :bug:`1631309`.
+
+ Creating :ref:`compressed_columns` in |MyISAM| tables by specifying partition
+ engines would not result in error. Bug fixed :bug:`1631954`.
+
+ It was not possible to configure basedir as a symlink. Bug fixed
+ :bug:`1639735`.
+
+ Slave did not report a ``Seconds_Behind_Master`` with multiple number of slave
```

Comment:
Replication slave did not report SBM correctly when running in multi-threaded slave mode